### PR TITLE
Add mutation effect in the annotated maf, fusions, and cna files

### PR DESCRIPTION
**Pull request by zhx828**
This solves issue #16

### CODE REVIEW

File: AnnotatorCore.py [hunk 1]
```diff
@@ -759,7 +762,8 @@ def cacheannotated(annotatedfile, defaultCancerType, cancerTypeMap):
             istart = geIndexOfHeader(headers, ['PROTEIN_START'])
             iend = geIndexOfHeader(headers, ['PROTEIN_END'])
             icancertype = geIndexOfHeader(headers, ['ONCOTREE_CODE', 'CANCER_TYPE'])
-            # imutationeffect = headers['MUTATION_EFFECT']
+            imutationeffect = headers['MUTATION_EFFECT']
+            imutationeffectcitations = headers['MUTATION_EFFECT_CITATIONS']
```
Comment (jjgao) [inline AnnotatorCore.py]: Maybe we can all the citations including the therapeutic ones?

File: AnnotatorCore.py [hunk 2]
```diff
@@ -854,20 +859,26 @@ def pulloncokb(hugo, proteinchange, alterationtype, consequence, start, end, can
...
+            if(evidences['mutationEffect'] is not None):
+                oncokbdata['mutation_effect'] = evidences['mutationEffect']['knownEffect']
+
+                # mutation effect citations
+                for pmid in evidences['mutationEffect']['citations']['pmids']:
+                    oncokbdata['mutation_effect_citations'].append(pmid)
+
+                for abstract in evidences['mutationEffect']['citations']['abstracts']:
+                    abstractStr = abstract['abstract'] + '(' + abstract['link'] + ')'
+                    oncokbdata['mutation_effect_citations'].append(abstractStr)
```
Comment (jjgao) [inline AnnotatorCore.py]: maybe call it `citations` now.

**Comment by zhx828** [comment]
Sounds good

**Comment by zhx828** [comment]
@jjgao could you take a look? I included the citations column at the end of the file

**Pull request merged**